google/resource_google_project_test.go

```diff
@@ -402,6 +459,15 @@ resource "google_folder" "folder1" {
 `, pid, projectName, folderName, org)
 }
 
+func testAccProject_appEngineNoApp(pid, org string) string {
```

Comment:
this is pretty much identical to `testAccProject_create` (which lives in `resource_google_project_iam_policy_test.go`) if you wanted to reuse that (a few other resources are doing so)

Reply:
I have a weakly-held opinion that it's better to have the configs for semantically different things separated, even if the implementation ends up looking the same, to minimize unintended consequences of changes. But if you feel strongly about it, I'm happy to change this.

Reply:
Cool that's fine!